docs/prerequisites.md

```diff
@@ -20,55 +19,52 @@
     --from-file=credentials=credentials
     ```
   
-3. Alternatively use the *CredentialsRequest* resource in the `hack` directory
+2. Alternatively use the `CredentialsRequest` resource in the `hack` directory:
 
    ```bash
    oc apply -f https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
    ```
 
 ### For STS clusters
 
-The same steps mentioned above can be followed for STS clusters as well.
-But if instead of manually creating the secret in `step 2`, you prefer
-`step 3`, then there are additional steps involved.
-
-1. After creating the *CredentialRequests*,
-
-    ```bash
-    oc get credentialsrequest -n openshift-cloud-credential-operator  \
-        aws-load-balancer-controller -o yaml > <path-to-credrequests-dir>/cr.yaml
-    ```
-
-    Extract and save the required *CredentialsRequest* in a directory.
-
-2. Use the ccoctl tool to process all *CredentialsRequest* objects in the previously specified
-directory:
+1. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects needed to bootstrap the operator:
 
     ```bash
+    curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
     ccoctl aws create-iam-roles \
         --name <name> --region=<aws_region> \
         --credentials-requests-dir=<path-to-credrequests-dir> \
         --identity-provider-arn <oidc-arn>
     ```
 
-    For each *CredentialsRequest* object, `ccoctl` creates an IAM role with a trust
+    For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust
     policy that is tied to the specified OIDC identity provider, and permissions
-    policy as defined in each *CredentialsRequest* object. This also generates a set
+    policy as defined in each `CredentialsRequest` object. This also generates a set
     of secrets in a **manifests** directory that is required
     by the **aws-load-balancer-operator**.
 
-    **Note**: To Extract and prepare the `ccoctl` binary, documentation can be
-    found [here](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts).
+    **Note**: To extract and prepare the `ccoctl` binary, documentation can be
+    found [here](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts).
 
-3. Apply the secrets to your cluster:
+2. Apply the secrets to your cluster:
 
     ```bash
     ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
     ```
```

Comment:
These look like essentially the same instructions as you are adding in `install.md`.  The only substantive difference that I see is that the instructions in `install.md` use the CredentialsRequest that the operator has created in the cluster whereas these instructions (in `prerequisites.md`) use the CredentialsRequest manifest from GitHub.  Am I missing anything?  Could you combine the instructions and simply tell the reader either (a) to get the manifest from GitHub or (b) to install the operator and then to get the CredentialsRequest from the cluster?

Reply:
Yes, you are right, the instructions are very similar, in the way they use `ccoctl` and created the generated secrets. However they differ in some steps, most importantly step1 from `install.md` which explains that `CredentialsRequest` is created for `AWSLoadBalancerController` CR.
I saw them similar but still not identical, so decided to not force the refactoring of the common part.